staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_testing_utils_test.go

```diff
@@ -117,12 +117,16 @@ func compactStorage(c *CacheDelegator, client *clientv3.Client) storagetesting.C
 			c.cacher.watchCache.startIndex++
 		}
 		c.cacher.watchCache.listResourceVersion = rv
+		if _, err := client.Compact(ctx, int64(rv)); err != nil {
+			t.Fatalf("Could not compact: %v", err)
+		}
+	}
+}
 
-		if _, err = client.KV.Put(ctx, "compact_rev_key", resourceVersion); err != nil {
+func increaseRV(client *clientv3.Client) storagetesting.IncreaseRVFunc {
+	return func(ctx context.Context, t *testing.T) {
+		if _, err := client.KV.Put(ctx, "compact_rev_key", "123"); err != nil {
```

Comment:
Should we change that to some random other key (instead of using compact_rev_key that has a potential to mess something up)?